h2o-py/h2o/backend/__init__.py

```diff
@@ -41,22 +41,10 @@
 from distutils.version import StrictVersion
 import sys
 
-import colorama
 
 from .cluster import H2OCluster
 from .server import H2OLocalServer
 from .connection import H2OConnection
 from .connection import H2OConnectionConf
 
-if not hasattr(colorama, '__version__'):
-    print("[WARNING] H2O requires colorama module of version 0.3.8 or newer. You have older version .\n"
-          "You can upgrade to the newest version of the module running from the command line\n"
-          "    $ pip%s install --upgrade colorama" % (sys.version_info[0]))
-    sys.exit(1)
-elif StrictVersion(colorama.__version__) < StrictVersion("0.3.8"):
-    print("[WARNING] H2O requires colorama module of version 0.3.8 or newer. You have version %s.\n"
-          "You can upgrade to the newest version of the module running from the command line\n"
-          "    $ pip%s install --upgrade colorama" % (colorama.__version__, sys.version_info[0]))
-    sys.exit(1)
-    
```

Comment:
found this very ironic that a library used to print in the client console was checked in the `backend` module…